### ISSUE TITLE
Feature request: resource transparency on tiles under fog of war

### ISSUE THREAD

**Issue by Cwpute** [yairm210/Unciv#10494]
**Is your feature request related to a problem? Please describe.**
Resources icons on tiles under fog of war effect clearly have an effect put on them, but it has currently no purpose: because it is not strong enough, it does not allow you to focus on discovered tiles only, but because there's still an effect, it gives you the impression that these ressources are different.

**Describe the solution you'd like**
Apply a stronger transparency on them so as to really tell us that these are currently out of our vision field.

**Describe alternatives you've considered**
Remove the transparency altogether.

**Additional context**
I am colourblind and am especially attentive to contrasting interfaces that can clearly convey what they want to convey. So yeah 🤡

**Comment by yairm210** [comment]
![image](https://github.com/yairm210/Unciv/assets/8366208/fc2190a0-26b6-44c8-9a72-217b8f3e9d48)
Done, thanks

**Comment by Cwpute** [comment]
Thanks, but actualy, i was talking about _these_ ressources 😂 :
![resources tranparency](https://github.com/yairm210/Unciv/assets/80604512/6c5f50a0-5efc-4bbf-9f63-33b43d02e25b)

I don't know why the image didn't upload the first time around.
Here i can clearly see the Industry points have changed, but i'm havin a hard seeing the same difference for Food and Gold.

**Comment by yairm210** [comment]
You mean *yields*! That's 100% intentional - yield view is not meant to be permanently on, because it's generally distracting.
If you're getting distracted by yields you should probably turn them off.
But we can poll this as well

**Comment by Cwpute** [comment]
Ok, _yelds_, sorry :) i don't have the whole vocabulary for this game yet.
Let me rephrase the whole thing then.

So the transparency on yelds outside your vision is not strong enough imo.
My guess is transparency was applied to them so as to clearly show they were curently out of reach, but since it is not so strong, it just appears as… washed out, but does not clearly convey it is out of reach (if that was the intent?)

What i was distracted by i think is the fact i clearly saw the yelds were a bit different outside my vision field, but didn't understood why exactly.

**Comment by Cwpute** [comment]
> That's 100% intentional

So do you mean this partial, not clearly defined transparency was intentional ?
What's its purpose then ? If the transparency is not supposed to be visible, what's the point of having transparency in the first place?
Either entirely remove it, then if yelds are too annoying they can be switched off, or strenghten it, as to give ita purpose.

**Comment by yairm210** [comment]
There's no transparency on the yields
What there is is the *grainy overlay* on the tiles for visual effect, that's probably what you're seeing (I don't see it on the yields, personally)

**Comment by Cwpute** [comment]
I'm not talking about the grainy overlay.
Here is a side by side compariswn of the yelds from my previous screenshot: 
![side by side comparison](https://github.com/yairm210/Unciv/assets/80604512/d84b700e-1087-4378-be0c-992cdf0de10e)

**Comment by Cwpute** [comment]
I thunk it's much more visible on the Industry yelds: 
![resources tranparency2](https://github.com/yairm210/Unciv/assets/80604512/8e162749-c4a5-4b3a-b82d-113e16d6dd24)
